src/MenuConfig.cpp

```diff
@@ -287,6 +291,28 @@ MenuConfig::MenuConfig (bool _is_game_state)
 	minimap_lstb->append(msg->get("Visible (2x zoom)"), "");
 	minimap_lstb->append(msg->get("Hidden"), "");
 
+	// set up low hp notification type combinantions
+	low_hp_warning_lstb->append(msg->get("Disabled"), "Disables all low health notifications.");
```

Comment:
To reduce translation load, maybe these should have one tooltip string that describes the option as a whole. For example:

>When the player's health is low, they can be notified through a message, sound, and/or cursor change.

Also, it should be wrapped with `msg->get()` to ensure it gets translated.

**But, I would say hold off on changing anything here. I have some more thoughts on which options should be available.**